### ISSUE TITLE
Fix field labels

### ISSUE THREAD

**Issue by devwander** [EvertnRS/even4#1]
In the program's forms, input fields lack consistent and accessible labels, which can impact usability and accessibility for users, especially those using screen readers. To ensure a seamless experience, all fields should have appropriate labels that are either visible or screen reader-friendly.

![image](https://github.com/user-attachments/assets/2fa37082-e1d7-4ad9-82c9-df4508805fa7)
